globus_sdk/config/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .env_vars import get_environment_name, get_http_timeout, get_ssl_verify
+from .environments import get_config_by_name, get_service_url
+
+__all__ = (
+    "get_config_by_name",
+    "get_service_url",
+    "get_environment_name",
+    "get_ssl_verify",
+    "get_http_timeout",
+)
```

Comment:
Moving from a module to a subpackage gives us a cleaner definition of the interface.
The only one of these which isn't strictly needed is `get_config_by_name` (since it's wrapped by `get_service_url`).

Should I add `EnvConfig` to the list as well? I waffled on that. I could be convinced either way.

Reply:
EnvConfig moved into here and `get_config_by_name` became `EnvConfig.get_by_name`